multi-int-store/src/main/resources/application.yml

```diff
@@ -1 +1,12 @@
-callback-accept-version: 0.1.0-SNAPSHOT
\ No newline at end of file
+callback-accept-version: 0.1.0-SNAPSHOT
+logging:
+  level:
+    root: ERROR
+    org.springframework.web.filter.CommonsRequestLoggingFilter: DEBUG
+    com.connexta: DEBUG
```

Comment:
❗️ I no longer see logs from the com.connexta packages

Reply:
With latest changes, you should be able to see them:

`cdr_local-minio_multi-int-store.1.sfryp89wwq5k@aaron-dell    | 2019-07-06 19:21:49.656 ERROR 22 --- [nio-8080-exec-1] c.c.m.controllers.RetrieveController     : TESTING......`